### PR TITLE
Working graphql connection to Risk and Client backend

### PR DESCRIPTION
**Pull request by luisgoncalvesanchor**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,17 +1,42 @@
...
-  webpack: config => {
+  webpack: (config, { isServer } ) => {
     config.module.rules.push({
       test: /\.(graphql|gql)$/,
       exclude: /node_modules/,
       loader: 'graphql-tag/loader',
     });
+    if (!isServer) {
+      config.resolve.fallback.fs = false;
+    }
```
Comment (volkan-anchor) [inline next.config.js]: Lets add very descriptive comments in our config files for future references. This is going to be a well maintained into the future

File: package.json [hunk 2]
```diff
@@ -32,5 +36,9 @@
       "\\.(gql|graphql)$": "jest-transform-graphql",
       ".*": "babel-jest"
     }
+  },
+  "browser": {
+    "child_process": false,
+    "fs": false
```
Comment (volkan-anchor) [inline package.json]: Can you comment on why we need this?

File: server.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+  server.all('*', (req, res) => {
+    return handle(req, res)
+  })
+
+  server.listen(port, (err) => {
+    if (err) throw err
+    console.log(`> Ready on http://localhost:${port}`)
+  })
+})
```
Comment (volkan-anchor) [inline server.js]: Let's convert this into TS.

File: server.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+app.prepare().then(async () => {
+  const server = express()
+
+  const keyLocation =
+  process.env.HOME + '/.config/dotfiles/frontend-tester-key.json';
+  if (!fs.existsSync(keyLocation)) {
+    execSync(
+      'gcloud iam service-accounts keys create --iam-account frontend-tester@development-204920.iam.gserviceaccount.com ' +
+        keyLocation
+    );
```
Comment (volkan-anchor) [inline server.js]: I feel like this should be in our onboarding session. And this should just throw.

File: src/currentUser/currentUser.graphql [hunk 5]
```diff
@@ -0,0 +1,5 @@
+query CurrentUser {
+	currentUser {
```
Comment (volkan-anchor) [inline src/currentUser/currentUser.graphql]: spacing?

Comment (volkan-anchor) [reply src/currentUser/currentUser.graphql]: does your editor autofix this? @luisgoncalvesanchor

File: src/vaults/vaults.graphql [hunk 6]
```diff
@@ -0,0 +1,6 @@
+query Vaults {
+	vaults {
```
Comment (volkan-anchor) [inline src/vaults/vaults.graphql]: same here

**Review by volkan-anchor — CHANGES_REQUESTED** [review]

**Comment by volkan-anchor** [comment]
Can we also add a documentation of all the things you've done in `.steps.md` so we can look back and repeat the process.

File: server.js (hunk 4 above)
Comment (luisgoncalvesanchor) [reply server.js]: This was copied from setupProxy.js file from Risk Dashboard

**Review by volkan-anchor — APPROVED** [review]

**Pull request merged**